src/types/account.js

```diff
@@ -215,7 +220,8 @@ export type AccountRaw = {
   lastSyncDate: string,
   endpointConfig?: ?string,
   subAccounts?: SubAccountRaw[],
-  balanceHistory?: BalanceHistoryRawMap
+  balanceHistory?: BalanceHistoryRawMap,
+  resources?: ResourcesLike
```

Comment:
by introducing a new field in Account, you will also need to add it in `patchAccount` otherwise desktop will not see any update of that field (desktop is using this reconciliation logic to minimize the object updates due to fact we run things in different threads)